src/powfacpy/plot_interface.py

```diff
@@ -224,6 +224,37 @@ def set_x_axis_range_of_active_plot(self,range: Iterable):
     x_axis = self.get_x_axis_of_active_plot()
     x_axis.rangeMin = range[0]
     x_axis.rangeMax = range[1]
+
+  def plot_from_comtrade(self,
+                      file_path,
+                      variables,
+                      graphics_page=None,
+                      plot=None,
+                      parent_folder_comtrade=None,
+                      **kwargs):
+    """Plot a varibale from a COMTRADE formated file under file_path (*.cfg).
+    Creates the comtrade object (IntComtrade) and plots.
+    
+    If you want to plot from a comtrade object (IntComtrade) that already 
+    exists in the PF database, use the method plot_monitored_variables as shown.
```

Comment:
what do you want to say with 'as shown'? maybe 'as in the code below' or does this refer to other docs?

Reply:
I added "as shown in this method"